content/docs/12.5-palette-cli/20-commands.md

```diff
@@ -149,7 +163,9 @@ To learn more about installing a PCG cluster. Refer to each platform's respectiv
 
 # PDE
 
-The `pde` subcommand interacts with the Palette Dev Engine (PDE) platform and its resources. You can use the `pde` command to log in to Palette, manage virtual clusters, and switch the project scope. The `pde` command exposes the following subcommands.
+The `pde` subcommand interacts with the Palette Dev Engine (PDE) platform and its resources. You can use the `pde` command to log in to Palette, manage virtual clusters, and switch the project scope. 
+
+The `pde` command exposes the following subcommands.
```

Comment:
```suggestion
The `pde` command exposes the following subcommand.
```

Reply:
Since there's only one subcommand, it feels awkward to use plural to introduce it.